superset-frontend/src/explore/components/controls/SelectAsyncControl/SelectAsyncControl.test.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import SelectAsyncControl from '.';
+
+jest.mock('src/components/AsyncSelect', () => ({
+  __esModule: true,
+  default: (props: any) => (
+    <div
+      data-test="select-test"
+      data-data-endpoint={props.dataEndpoint}
+      data-value={JSON.stringify(props.value)}
+      data-placeholder={props.placeholder}
+      data-multi={props.multi ? 'true' : 'false'}
+    >
+      <button
+        type="button"
+        onClick={() => props.onChange(props.multi ? [] : {})}
+      >
+        onChange
+      </button>
+      <button type="button" onClick={() => props.mutator()}>
+        mutator
+      </button>
+
+      <div data-test="valueRenderer">
+        {props.valueRenderer({ label: 'valueRenderer' })}
+      </div>
+    </div>
+  ),
+}));
+
+let defaultProps = {
```

Comment:
`beforeEach` is already initializing this variable.

`let defaultProps;` or `let defaultProps = {};`

Reply:
For typing. If don't put it that way, the type will be `any` or `{}`.

Reply:
done by factory: https://github.com/apache/superset/pull/13753#discussion_r600862661